src/scripts/colors.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
+
+class bcolors:
+    HEADER = '\033[95m'
```

Comment:
Since it is a "helper" class follow the next syntax:

```
class _blcolors:
  header = ...
```

So the class name is underscored, and use lowercase characters for the members.

Reply:
fixed